### ISSUE TITLE
Cannot import exported file

### ISSUE THREAD

**Issue by nikola-benes** [sybila/biodivine-aeon-client#17]
The following file (created via export) cannot be imported:

```
#position:P53:487.10138075559905,210.04371654758893
M2N -| P53
#position:DNA:695.4721342882492,213.44620915677535
P53 -| DNA
#position:M2C:484.0876581001654,374.27432908939556
P53 -> M2C
#position:M2N:696.577127528888,379.0788979596962
$M2N:(!DNA | M2C) & g(P53)
DNA -| M2N
P53 -| M2N
M2C -> M2N
#position:v_5:-3.0691224539378376e+39,-2.272417347981254e+39
```

The problem is in the last line. It seems that I have accidentally created a vertex too far from the others. Anyway, the tool exported the model just fine but is unable to import it again.

**Comment by daemontus** [comment]
Wow, that is indeed quite far away :D The problem is probably because some parser in the pipeline (I'm guessing Rust, JS is probably ok with this) does not understand FP scientific notation... will look into it.

**Comment by daemontus** [comment]
Will be fixed in `0.4.0`. The details are in 73edf0fa46e6b5cdcf91490701424c7b1763c89d. Similar fix is also implemented in the server repo (see [3198722](https://github.com/sybila/biodivine-aeon-server/commit/31987229e9d96f94f75200d492f52f00f1172969)).
